papermill/execute.py

```diff
@@ -151,8 +147,6 @@ def execute_notebook(notebook,
         progress_bar (bool): Flag for whether or not to show the progress bar.
         log_output (bool): Flag for whether or not to write notebook output to stderr.
     """
-    print("Input Notebook:  %s" % get_pretty_path(notebook))
```

Comment:
Please leave these in, you can change get_pretty_path to handle None instead

Reply:
This is the same about dealing with the output file.

Reply:
From your [comment](https://github.com/nteract/papermill/pull/107#issuecomment-360376791) I don't think you need to remove this. This generates some output on stdout but that doesn't get in the way of what you are trying to do.